docs/quick-start.md

```diff
@@ -42,7 +42,8 @@ Create the following files:
 ### `src/routes/__root.tsx`
 
 ```tsx
-import { RootRoute } from '@tanstack/react-router'
+import { RootRoute, Link, Outlet } from '@tanstack/react-router'
+import { TanStackRouterDevtools } from "@tanstack/router-devtools"
```

Comment:
please use single quotes everywhere

Reply:
Sure! Let me change that

Reply:
replaced with single quotes while resolving conflicts from the main branch in the latest commit 160b849